css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,39 @@
+body {
+  font-family: DM Sans, sans-serif;
+}
+
+h1 {
+  color: black;
+  font-family: DM Sans, sans-serif;
+}
+
+span {
+  color: rgb(210, 210, 207);
+}
+
+.images {
+  display: flex;
+  flex-direction: row;
+}
+
+.column {
+  flex-basis: calc(100% / 5);
+}
+
+.title {
+  text-align: center;
+}
+
+img {
+  width: 200px;
+  height: 375.3px;
```

Comment:
Be careful when you set the image dimensions like this. Your images are warped, so you will need to either remove the height or add `object-fit`